src/main/resources/viewSql/patientAppointmentView.sql

```diff
@@ -2,8 +2,8 @@ SELECT
   pd.person_id AS patient_id,
   pd.gender,
   pd.birthdate               AS birth_date,
-  EXTRACT(YEAR FROM (SELECT age( pap.appointment_start_time, pd.birthdate))) AS age_at_appointment,
-  age_group(pap.appointment_start_time, pd.birthdate) As age_group_at_appointment,
+  EXTRACT(YEAR FROM (SELECT age( pap.appointment_start_time, TO_DATE(CONCAT('01-01-', pd.birthyear), 'dd-MM-yyyy')))) AS age_at_appointment,
```

Comment:
Shall we use the built-in function to create TIMESTAMP? Same for other views also.
Eg: `SELECT EXTRACT (YEAR FROM (SELECT age(to_timestamp('25-04-2020', 'DD-MM-YYYY'), to_timestamp('2010', 'YYYY')))) AS X;`

Reply:
Done